packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -433,6 +433,62 @@ void main() {
     },
   );
 
+  testWidgets('Text field drops selection color when losing focus', (WidgetTester tester) async {
```

Comment:
Nit: Is it relevant to say "Regression test for https://github.com/flutter/flutter/issues/103341" here and elsewhere?

Reply:
> LGTM 👍
> 
> Just as a thought, what if we passed a transparent Color as selectionColor instead of null?

That works too. Though if someone explicitly passes `null` as `selectionColor` to `EditableText`. I think they would expect no selection to be painted, where as now it defaults to the color defined by `DefaultSelectionStyle.of(context)`.

Reply:
I don't have a strong opinion on this, but if I were to redesign this again, passing null as color means disabling the selection seems to be a weird API

Reply:
I agree, though another small difference between `null` and `Colors.transparent` is that the TextHighlightPainter skips painting all together when the `selectionColor` is null vs when it is `Colors.transparent` it will still paint some values. https://github.com/flutter/flutter/blob/df7111a848cb0cc076a9c89d48332f0ec2700c45/packages/flutter/lib/src/rendering/editable.dart#L2726